binary-option/README.md

```diff
@@ -12,6 +12,10 @@ Now suppose the Bucks win Game 3, and the estimated probability of the Bucks win
 
 We'll discuss this mechanism in more detail later.
 
+## Note
+
+This code is unaudited. Use at your own risk.
```

Comment:
legal approved? :sweat_smile:

Reply:
They requested some changes to surface audit information directly in the README. And probably some other stuff later, but this should cover the first round of changes.